### DISCUSSION TITLE
787-8 Wing Inspection Light Source

### DISCUSSION THREAD

**Discussion by cajunjosh** [magknight/787aviatorsTracker#796] [Completed things] [answered]
Could we get the wing inspection light sources moved from their current position back to the correct one?

![Wing Inspection Lights](https://user-images.githubusercontent.com/75854551/162646151-b88744ab-b6df-40ad-bd83-f86125f59f03.PNG)

**Comment by aeroniemi** [comment, marked as answer]
duplicate of #794, was fixed in 1.7.46
